pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -666,14 +666,18 @@ class PagerState(
                 listOf<Any>(
                     it.pageCount,
                     it.currentPage,
+                    it.currentPageOffset,
```

Comment:
We explicitly do not want the offset to be saved.

Reply:
Does `offscreenLimit `need to be saved?

Reply:
I don't think so. The value is required to be set at init, and the value rarely changes.

Reply:
When navigating to a new page and then returning, the value of `PagerState.offscreenLimit` will always set to 1.  Just like the video of #545

Reply:
Ah ok, let's keep that then!